tests/rbd_mirror/test_rbd_clone_mirror.py

```diff
@@ -0,0 +1,111 @@
+       conf and keyring files on each node.
+
+    Test Case Flow:
+    1. Create a pool on both clusters.
+    2. Create an Image on cluster 1 and create a clone of the image in same pool.
+    3. Configure mirroring (peer bootstrap) between two clusters.
+    4. Enable pool mood journal based mirroring on the pool respectively.
+    5. Verify for the clone image is also get mirrored.
+    6. Create one more clone from the image and verify it is mirrored.
+    """
+
+    try:
+        log.info("Running RBD mirroring test case for clone images")
+        config = kw.get("config")
+        rbd = Rbd(**kw)
+
+        poolname = rbd.random_string() + "9521pool"
+        imagename = rbd.random_string() + "9521image"
+        snap = rbd.random_string()
+        clone1 = rbd.random_string()
+        size = "10G"
+        image_feature = "exclusive-lock,layering,journaling"
+        # create pool and image on cluster1
+        size = kw.get("size", "10G")
+        if not rbd.create_pool(poolname=poolname):
+            log.error(f"Pool creation failed for pool {poolname}")
+            return 1
+        rbd.create_image(
+            pool_name=poolname,
+            image_name=imagename,
+            size=size,
+            image_feature=image_feature,
+        )
+
+        # create clone of the image
+        rbd.snap_create(poolname, imagename, snap)
+        snap_name = f"{poolname}/{imagename}@{snap}"
+        rbd.create_clone(
+            snap_name=snap_name,
+            pool_name=poolname,
+            image_name=clone1,
+            clone_version="v2",
+        )
+
+        # enable journal based mirroring for pool mode
+        mirror1, mirror2 = [
+            rbdmirror.RbdMirror(cluster, config)
+            for cluster in kw.get("ceph_cluster_dict").values()
+        ]
+        imagespec = poolname + "/" + imagename
+        mirror2.create_pool(poolname=poolname)
+        mirror1.config_mirror(mirror2, poolname=poolname, mode="pool")
+        mirror2.wait_for_status(poolname=poolname, images_pattern=2)
+        mirror1.wait_for_status(imagespec=imagespec, state_pattern="up+stopped")
+        mirror2.wait_for_status(imagespec=imagespec, state_pattern="up+replaying")
+
+        # verify for cloned image mirrored
+        imagespec1 = poolname + "/" + clone1
+        mirror2.wait_for_status(poolname=poolname, images_pattern=2)
+        mirror1.wait_for_status(imagespec=imagespec1, state_pattern="up+stopped")
+        mirror2.wait_for_status(imagespec=imagespec1, state_pattern="up+replaying")
+
+        # Create clone of the image and verify for the image mirrored status
+        clone2 = rbd.random_string()
+        imagespec2 = poolname + "/" + clone2
+        rbd.create_clone(snap_name, poolname, clone2)
+        mirror2.wait_for_status(poolname=poolname, images_pattern=3)
+        mirror1.wait_for_status(imagespec=imagespec2, state_pattern="up+stopped")
+        mirror2.wait_for_status(imagespec=imagespec2, state_pattern="up+replaying")
```

Comment:
can we log clone mirrored verification for success and failure?

Reply:
added check for cloned mirror image in secondary